data/human/human missions.txt

```diff
@@ -7454,3 +7454,600 @@ mission "Adelita 5"
+				`		Howard Vendeur`
+				`		Investor Relations Manager, Megaparsec Incorporated`
+	on complete
+		event "10m investment done" 180
+		payment 25000
+		conversation
+			`You arrive back on Sunracer to a spaceport buzzing with activity. You make your way through the bustling shipyard toward the towering Megaparsec building. Entering the lobby, the receptionist gives you a nod, and you quickly ascend to the eighth floor.`
+			`	Howard greets you with a smile as you step into his office. The room is as you left it -- modern, modest, and with a clear view of the shipyards below. "Welcome back," he says, gesturing for you to take a seat. "I take it the job in Oblate went smoothly?"`
+			choice
+				`	"Of course."`
+					goto next1
+				`	"Didn't you message me as soon as I did it?"`
+			`	"True enough, just making conversation," he says with a grin. "The important thing is that the job's done. Now, let's get down to business.`
+			label next1
+			`	"Our recovery team is already retrieving the ship as we speak. You've done well. Here's a payment for that favor, 25,000 credits."`
+			choice
+				"Thanks."
+					goto investmentopps
+				"25,000?! You must have forgotten a zero there, you cheapskate!"
+					goto cheapskate
+				"I appreciate the offer, but there's no need for payment this time. Consider it purely a favor--I'm happy to help."
+			action
+				"syndicate business: mean to Howard" --
+				payment -25000
+			`	"That's generous of you, and I appreciate it."`
+				goto investmentopps
+			label cheapskate
+			`	"I see... well, let's not make this more difficult than it needs to be," he says in a clipped tone. "I'll transfer another 175,000 credits now to make it an even 200,000. But in the future, I'd appreciate it if you would remember the privilege you have of working with Megaparsec Incorporated.`
+			action
+				"syndicate business: mean to Howard" ++
+				payment 175000
+			label investmentopps
+			`	"Now, let's talk about that investment opportunity."`
+			`	Howard pulls up a holographic display on the digital screen behind him. Various charts and projections flash by. "Ready for the pitch?"`
+			choice
+				`	"You know it."`
+				`	"Of course."`
+			`	"Here's what we're looking at. Megaparsec Incorporated is expanding a refueling station on Crossroads, and we're likely going to go with a conventional expansion. But don't let the term 'conventional' fool you, Megaparsec rarely expands its refueling stations--this is the first time in over a decade. When we do, it's because we've identified a strategic location that will skyrocket profits. These opportunities are as rare as they come, and most investors never even hear about them. But you've proven yourself, and that's why you're getting a seat at the table."`
```

Comment:
```suggestion
			`	"Here's what we're looking at. Megaparsec Incorporated is expanding a refueling station on Crossroads, and we're likely going to go with a conventional expansion. But don't let the term 'conventional' fool you, because Megaparsec rarely expands its refueling stations--this is the first time in over a decade. When we do, it's because we've identified a strategic location that will skyrocket profits. These opportunities are as rare as they come, and most investors never even hear about them. But you've proven yourself, and that's why you're getting a seat at the table."`
```